### DISCUSSION TITLE
Aggregation statistics and equalization in MetViewer

### DISCUSSION THREAD

**Discussion by gsketefian** [dtcenter/METplus#2382] [Statistical Computation] [answered]
Hi,

The MetViewer documentation [here](https://metviewer.readthedocs.io/en/latest/Users_Guide/batch.html#batch-plotting-module) says:
```
If agg_stat is turned on, event equalization must be turned off (it is done automatically).
```
I am generating various verification metrics plots (AUC, bias, Brier Score, frequency bias, reliability, rank histogram, spread-skill) using MetViewer that compare two or more forecast models, and my team (at the DTC) had decided we wanted to turn on equalization for all metrics because we thought this would give us a fairer model comparison.  However, for some types of metrics, we also have aggregation statistics enabled.  From the documentation line I quoted above, it sounds like either equalization or the aggregation statistics need to be disabled.

Is that true, and if so, why does this restriction exist in MetViewer, i.e. what is it about aggregation statistics that makes event equalization not possible (or not make sense to calculate)?

Thanks for your help.

**Comment by TatianaBurek** [comment, marked as answer]
@gsketefian This is a outdated statement and should be removed. Aggregation statistic logic and event equalization can be used together

**Reply by gsketefian** [reply]
@TatianaBurek Oh ok, thanks!  I assume someone on the MET and/or MetViewer team will fix the documentation.
